components/Feature/Services/index.module.scss

```diff
@@ -24,3 +19,7 @@
   margin-top: 16px;
   clear: both;
 }
+
+.hidden {
+  display: none;
+}
```

Comment:
I think there's a `hidden` that we can use instead of defining our own class, it acts the same way that's described here (<div hidden={true}> or just <div hidden> should work, for conditional <div hidden={showOrHideBoolHere}>)

Reply:
Removed this css class. I didn't have to use hidden= as the showMore boolean covers this now, without need for css.